content/en/database_monitoring/setup_oracle/exadata.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Setting Up Database Monitoring for Oracle RAC
```

Comment:
The title of this article is the same as the other one. Was that intentional, or should this instead be "Setting Up Database Monitoring for Oracle Exadata"?